src/lazyload.js

```diff
@@ -9,7 +9,8 @@
 }(this, function() {
 
     var _defaultSettings,
-        _isInitialized = false;
+        _isInitialized = false,
+        _isFirstLoop = true;
```

Comment:
the _isFirstLoop variable cannot be a local variable shared with all instances of lazyload, it must be an instance variable instead. So it should be initialized with this._isFirstLoop in the constructor (LazyLoad function, rows 178 - 191).